client/containers/Pub/PubDocument/PubFileImport/MetadataEditor.tsx

```diff
@@ -247,7 +247,7 @@ const MetadataEditor = (props: MetadataEditorProps) => {
 		if (attributions && attributions.length) {
 			return (
 				<>
-					<h6>Authors</h6>
+					<h6>Contributors</h6>
```

Comment:
Import automatically adds folks as byline attributions, so we should change this to `Attributions`

Reply:
✔️ - makes sense that byline attribution would be the default on import. Thanks for catching that.